pom.xml

```diff
@@ -21,11 +21,11 @@
   <parent>
     <groupId>alien4cloud</groupId>
     <artifactId>alien4cloud-parent</artifactId>
-    <version>2.1.0</version>
+    <version>2.1.1-SNAPSHOT</version>
```

Comment:
Would need to reference an Alien4Cloud milestone, or new release instead of a snapshot, else the plugin fails to load with the error :
```
A released archive cannot depends on snapshots archives
```